### PR TITLE
improve wrapping behaviour

### PR DESCRIPTION
**Pull request by margaree**
https://rally1.rallydev.com/#/157196228032/detail/defect/193096650252

Now looks like:
screen shot 2018-01-26 at 11 02 16 am

### CODE REVIEW

**Comment by jstefaniuk-d2l** [comment]
Should there be any right padding in these? I always find it a bit bleh when text goes right to the border of bigger tiles like this

File: components/d2l-all-assessments-list-item.html [hunk 1]
```diff
@@ -91,6 +91,10 @@
 				align-items: center;
 			}
 
+			.info-container {
+				padding-right: 10px;
```
Comment (CodeBaboon) [inline components/d2l-all-assessments-list-item.html]: Should have an RTL flip for this also.

**Comment by ChrisKraljevicD2L** [comment]
I can test if there's no more review thingies

**Comment by ChrisKraljevicD2L** [comment]
Testing Checklist
Sanity Checks
- [ ] Verify that unit tests and spec tests have been written, if appropriate
- [ ] Verify that unit tests, spec tests, and linting are all passing
- [ ] Verify that no code review comments are outstanding
- [ ] Verify that the code is behind a feature flag

Acceptance/Functionality
- [ ] Check that the acceptance criteria of the story has been fulfilled
- [ ] Check that related functionality has not been severely affected (smoke test)
- [ ] (UI) Check that the functionality is the same across browsers and devices

Accessibility (UI)
- [ ] Verify that workflows can be completed via screen reader/keyboard
- [ ] Verify that workflows can be completed when the browser is zoomed to 200%
- [ ] Verify that there are no obvious colour contrast/combination problems

Internationalization (UI)
- [ ] Verify that lang terms are not hard coded
- [ ] Verify that workflows can be completed when RTL

**Pull request merged**